website/docs/r/app_engine_application.html.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: "google"
+page_title: "Google: google_app_engine_application"
+sidebar_current: "docs-google-app-engine-application"
+description: |-
+ Allows management of an App Engine application.
+---
+
+# google\_app_engine_application
+
+Allows creation and management of an App Engine application.
+
+~> App Engine applications cannot be deleted once they're created; you have to delete the
+   entire project to delete the application. Terraform will report the application has been
+   successfully deleted; this is a limitation of Terraform, and will go away in the future.
+   Terraform is not able to delete App Engine applications.
+
+## Example Usage
+
+```hcl
+resource "google_project" "my_project" {
+  name = "My Project"
+  project_id = "your-project-id"
+  org_id     = "1234567"
+}
+
+resource "google_app_engine_application" "app" {
+  project         = "${google_project.my_project.project_id}"
+  location_id     = "us-central'
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `location_id` - (Required) The [location](https://cloud.google.com/appengine/docs/locations)
+   to serve the app from.
+
+* `auth_domain` - (Optional) The domain to authenticate users with when using App Engine's User API.
+
+* `serving_status` - (Optional) The serving status of the app. Note that this can't be updated at the moment.
```

Comment:
the code makes it look like it can, is this sentence wrong or should the code be updated?